### PR TITLE
OVA-1.0.4 Release Notes

### PR DESCRIPTION
**Pull request by mukul-seagate11**
OVA-1.0.4 Release Notes

-----
[View rendered doc/RELEASENOTES/CHANGELOG-1.0.4.md](https://github.com/Seagate/cortx/blob/mukul-patch-6/doc/RELEASENOTES/CHANGELOG-1.0.4.md)

### CODE REVIEW

**Comment by mukul-seagate11** [comment]
@TechWriter-Mayur, please review the Release Notes?

**Comment by TechWriter-Mayur** [comment]
This looks good to me.

File: doc/RELEASENOTES/CHANGELOG-1.0.4.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# CORTX OVA 1.0.4
+
+This release notes includes new features and bug fixes added to the CORTX OVA 1.0.4.
+
+# Features
+
+ 1. On the About page, the serial number is displaying.
```
Comment (gauravchaudhari02) [inline doc/RELEASENOTES/CHANGELOG-1.0.4.md:7]: Can we change the sentence to below? As it is a first sentence in Release notes, it should be properly framed.
`Display serial number in About Page`.

**Review by gauravchaudhari02 — CHANGES_REQUESTED** [review]

**Comment by mukul-seagate11** [comment]
@All, waiting to get https://github.com/Seagate/cortx/pull/976 reviewed

File: doc/RELEASENOTES/CHANGELOG-1.0.4.md (hunk 1 above)
Comment (mukul-seagate11) [reply doc/RELEASENOTES/CHANGELOG-1.0.4.md]: good catch, its corrected now

Comment (gauravchaudhari02) [reply doc/RELEASENOTES/CHANGELOG-1.0.4.md]: looks like it is not refelcted yet.

Comment (TechWriter-Mayur) [reply doc/RELEASENOTES/CHANGELOG-1.0.4.md]: Please update the sentence to: 

`The serial number displays on the about page.`

**Review by TechWriter-Mayur — CHANGES_REQUESTED** [review]

**Review by mukul-seagate11 — COMMENTED** [review]
.

**Review by mukul-seagate11 — COMMENTED** [review]
.

**Comment by gauravchaudhari02** [comment]
> .

No change

**Review by mukul-seagate11 — COMMENTED** [review]
approve DCO

**Comment by mukul-seagate11** [comment]
Raising new PR as DCO not passing

**Pull request closed without merging**